Dockerfile

```diff
@@ -68,11 +68,12 @@ RUN echo "$PATH"
 RUN dotnet try jupyter install
 
 # Build extensions
-RUN dotnet publish ${HOME}/src/Microsoft.DotNet.Interactive.XPlot -c Release -o ${HOME}/Notebooks/extensions/Microsoft.DotNet.Interactive.XPlot
+RUN dotnet build ${HOME}/src/Microsoft.ML.DotNet.Interactive.Extensions -c Release 
```

Comment:
Don't really NEED to build and then pack. Can just `dotnet pack`, and it will do everything required.